components/SmallContactForm.php

```diff
@@ -357,7 +359,7 @@ public function onFormSend(){
               $recaptcha = new ReCaptcha(Settings::get('google_recaptcha_secret_key'));
             }
 
-            $response = $recaptcha->setExpectedHostname($_SERVER['SERVER_NAME'])->verify(post('g-recaptcha-response'), $_SERVER['REMOTE_ADDR']);
+            $response = $recaptcha->setExpectedHostname($this->validationReCaptchaServerName)->verify(post('g-recaptcha-response'), $_SERVER['REMOTE_ADDR']);
```

Comment:
This could have been more compact like
```
$response = $recaptcha->setExpectedHostname($_SERVER['SERVER_NAME'] == "127.0.0.1" ? "localhost" : $_SERVER['SERVER_NAME'])->verify(post('g-recaptcha-response'), $_SERVER['REMOTE_ADDR']);
```
but would have looked messy.